### DISCUSSION TITLE
How to use where and filter in GraphQL in one query?

### DISCUSSION THREAD

**Discussion by markrado123** [graphql/graphql-js#3716] [Q&A]
It is possible using in one query also where and filter.
At the moment it is my query:
```
  query Memories($id: ID!) {
      memories(where: { event: { id: $id } }) {
        id
        name
        story
      }
    }
```

I want add filter:
```
filter: {
    name: {
      anyofterms: "england"
    },
```

**Comment by markrado123** [comment]
Solved, please close.

**Comment by juffalow** [comment]
Hi @markrado123 ! You can check how other services done it, for example [GitHub API v4](https://docs.github.com/en/graphql/overview/about-the-graphql-api) is using GraphQL and you can check it live in their [Explorer](https://docs.github.com/en/graphql/overview/explorer).

If you want to see example with source code, you can check my project [Quotes](https://quotes.juffalow.com).
